examples/mf6/different_ways_to_regrid_models.py

```diff
@@ -94,8 +94,9 @@
 # in this example we'll regrid the npf package manually and the rest of the packages using default methods.
 
```

Comment:
I think this is a nice opportunity to explain something about the ``RegridderWeightsCache``! Please add one or two sentences what it does and why it is useful.

Reply:
done